### ISSUE TITLE
BUG: latest update breaks SQL server connections

### ISSUE THREAD

**Issue by EmanNo1** [beekeeper-studio/beekeeper-studio#1124]
Snap just updated to 3.3.1 and connections to SQL servers no longer work with a self-signed certificate error. Also, the setting for dark mode was re-set.

![Selection_010](https://user-images.githubusercontent.com/2874846/160905476-be3a38d7-fc29-46c9-beb6-1707f1792653.png)
 
Version Info
![Selection_011](https://user-images.githubusercontent.com/2874846/160906796-3063bb96-e12e-4ff5-89d0-258b9dc3b2d8.png)

**Comment by rathboma** [comment]
Bah. I'll get this fixed. I had to bump the sql server library due to security patches. Thanks for the bug report!

Funnily enough my integration tests pass just fine...

**Comment by rathboma** [comment]
The dark mode flag was changed to 'system' so it sync with the OS

**Comment by theOofDoctor** [comment]
@rathboma  this may be related to #1121. I was able to connect to SQL server if I enable SSL, but like 90% of the time I try to run a query I get an ECONNRESET error, which I've read elsewhere is due to SSL being enabled

**Comment by romulocraveiro** [comment]
Hi! I've faced a similar problem as above with EmanNo1: 
![image](https://user-images.githubusercontent.com/64331827/160922376-16aee7ce-8c43-4c5a-9bab-0d95af07759d.png)

**Comment by rathboma** [comment]
Hey all. I have a fix for this - The SQL Server library introduced a new config setting `trustServerCertificate`, so I've included this in the connection form now.

Checking this box will return functionality to normal.

Can someone here download the test build:

https://github.com/beekeeper-studio/beekeeper-studio/actions/runs/2078292053

and check the `Trust Server Certificate` box and test for me?

**Comment by rathboma** [comment]
@romulocraveiro I'm not sure your error is the same. Was your PSQL connection working in the previous version of Beekeeper Studio?

**Comment by nsl-94** [comment]
Hello .

I had the same problem with a connection to an SQL Server. 

With the checkbox selected iwith the test build, it connects to the SQL Server correctly.

Thank you
Nuno

**Comment by sunshineco** [comment]
I can confirm that checking the new `Trust Server Certificate` box allows connections to SQL Server to succeed.

**Comment by romulocraveiro** [comment]
Yes, @[rathboma](https://github.com/rathboma) I think you are right. And yes, my PSQL was working before the update, but in the end, I solved my problem just by updating my PSQL as well yesterday. Thanks!

**Comment by EmanNo1** [comment]
Thank you, @rathboma. I can connect to multiple SQL servers again.

PS. What is the protocol, is it my responsibility to close this bug since I have a "Close with comment" option?

**Comment by rathboma** [comment]
You're welcome to close issues that are fixed or no longer relevant to you! I'll close it this time (seeing as I'm here). Either way I'm not the type of person that gets offended by stuff like that, so feel free to use best judgement.
